fib.py

```diff
@@ -6,6 +6,7 @@
 Negative numbers should return None
 """
 def fibonacci(position):
+  if(position == 0) return 0
```

Comment:
1. please notice that there is one colon missing 
2. please notice that the return statement should break the line